Double/js/script.js

```diff
@@ -0,0 +1,134 @@
+const imgData = [
+    {
+      src: "../img/among.png",
+      name: "AmongUs",
+    },
+    {
+      src: "../img/glasses.png",
+      name: "Glasses",
+    },
+    {
+      src: "../img/mango.png",
+      name: "Mango",
+    },
+];
+
+const cardBack = "../img/font.png";
+
+const doubleImgArrayCopy =  imgData.concat(imgData);
+const doubleImgArray = doubleImgArrayCopy.map((a, i) => Object.assign({}, a));
+const imgArray = shuffle(doubleImgArray);
+
+const secondsCounter = document.getElementById('seconds-counter');
+const modal = document.querySelector("[data-modal-window]");
+const playerNameSubmit = document.querySelector(".playerName-button");
+const cardField = document.getElementById("cardField"); 
+
+let matchedCardsCounter = 0;
+let currentCard = null;
+let resultsObj = {};
+let seconds = 0;
+
+function modalWindow(endTime) {
+    modal.style.display = "block";
+    playerNameSubmit.addEventListener("click", function(event){
```

Comment:
Add event on page load. Try to avoid adding them in functions which will be called later